doc/snippets/Microsoft.Data.SqlClient/SqlConnection.xml

```diff
@@ -1055,6 +1055,20 @@ GO
  ]]></format>
             </remarks>
         </RetrieveStatistics>
+        <RetrieveInternalInfo>
+            <summary>Returns a name value pair collection of internal properties at the point in time the method is called.</summary>
+            <returns>Returns a reference of type <see cref="T:System.Collections.Generic.IDictionary" /> of (string, object) items.</returns>
+            <remarks>
+            <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ When this method is called, the values retrieved are those at the current point in time. If you continue using the connection, the values are incorrect. You need to re-execute the method to obtain the most current values.
+ 
+ Supported internal properties: `SQLDNSCachingSupportedState` and `SQLDNSCachingSupportedStateBeforeRedirect`.
```

Comment:
Can you also describe about these properties what they mean and what info they provide. Maybe create a table below?

Reply:
Updated